dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -120,6 +122,32 @@ def cleanLength(length):
             raise ValueError("Length must be positive")
         return length
 
+    @staticmethod
+    def cleanRow(row):
+        for key in row.keys():
+            value = row[key]
+            if value is not None:
```

Comment:
I wouldn't think it matters. But with that in mind, is there any possibility of any field containing a list of numbers? That would throw this implementation off. It may just be safer to check the field type. What do you think?